zulip/zulip/__init__.py

```diff
@@ -1133,16 +1133,19 @@ def update_user_by_id(self, user_id: int, **request: Any) -> Dict[str, Any]:
             request=request
         )
 
-    def get_members(self, request: Optional[Dict[str, Any]] = None) -> Dict[str, Any]:
+    def get_users(self, request: Optional[Dict[str, Any]] = None) -> Dict[str, Any]:
         '''
-            See examples/list-members for example usage.
+            See examples/list-users for example usage.
         '''
         return self.call_endpoint(
             url='users',
             method='GET',
             request=request,
         )
 
+    def get_members(self, request: Optional[Dict[str, Any]] = None) -> Dict[str, Any]:
+        return self.get_users(request=request)
```

Comment:
Let's add a comment that this is a backwards-compatibility name.  I can take care of it before merging.